src/components/Checkbox/Checkbox.module.scss

```diff
@@ -0,0 +1,137 @@
+@import '../../styles/colors';
+
+button {
+  all: unset;
+}
+
+.Container {
+  display: flex;
+  align-items: center;
+  padding: 4px;
+}
+
+.Container:focus-within {
+  border-radius: 8px;
+  outline: 1px solid #{$green-11};
```

Comment:
nit: I don't think we need to use `#{}` to wrap color variables here. Should be able to just do `$green-11`. Don't need to change, just something to keep in mind :)

Reply:
removed the `#{}` for the styles inside the class. But for the variable, we have to use the `#{}`.